english/jacket/complete-jacket/ad.md

```diff
@@ -0,0 +1,3 @@
+You craft the greatest marshmallow jacket the world has ever seen. You're so excited to show your family and friends this life changing achivement. You step outside and a little boy runs up to you and asks "Michelin Man, can I have your autograph?"
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.